core/templates/domain/entity_editor_browser_tabs_info/entity-editor-browser-tabs-info.model.ts

```diff
@@ -0,0 +1,115 @@
+// Copyright 2022 The Oppia Authors. All Rights Reserved.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS-IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+/**
+ * @fileoverview Model for creating and mutating instances of entity editor
+ * browser tabs info.
+ */
+
+export interface EntityEditorBrowserTabsInfoObject {
+  entityType: string;
+  id: string;
+  latestVersion: number;
+  numberOfOpenedTabs: number;
+  someTabHasUnsavedChanges: boolean;
+}
+
+export class EntityEditorBrowserTabsInfo {
+  // Id of the entity whose editor page is opened.
```

Comment:
This doesn't seem to match the line of code that it's describing.

I would also suggest you include docstrings for all properties, and perhaps some general explanation of what a EntityEditorBrowserTabsInfo is (and is for). It is fine to write multi-paragraph-long explanations; just make sure the context is crystal-clear to a new developer.